### DISCUSSION TITLE
Results for the examples provided (SARS-CoV-2 main protease)

### DISCUSSION THREAD

**Discussion by Shubhangi1397** [Valdes-Tresanco-MS/gmx_MMPBSA#439] [General]
Hi everyone,
I have recently installed gmx|_MMPBSA to predict the binding free energy from GROMACS output files. I recently ran the test examples as per the documentations provided. 

I was wondering if there **are results that are available for the examples particularly the SARS-CoV-2 main protease example** (protein-ligand comples). I got the interaction energy graphs but don't really know if they are right as per the tutorial. 
 
Many thanks. 

Shubhangi

**Comment by marioernestovaldes** [comment]
Results are not available at the time although we plan to include the output files in the tutorials at some point... Anyway, the tutorial is designed to show that gmx_MMPBSA works and can process complex systems...
